content/spin/kv-store.md

```diff
@@ -0,0 +1,405 @@
+
+However, from Spin v0.9.0 onwards, you are no longer required to install any on-disk persistence; outside of just installing Spin itself. SQLite exists inside Spin. SQLite is embedded in a way that is analogous to how one would imagine using a software library i.e. SQLite is designed to be used in this manner and therefore, for all intents and purposes, you could say that SQLite does not exist outside of Spin. When you run your Spin application using the [spin up](https://developer.fermyon.com/common/cli-reference#up) command SQLite is available to your application. When your application is no longer running, neither is SQLite. Your data will continue to persist at all times (including during restarts) and will support the running of your application. Let's get started with creating and deploying your first Spin application that uses this new key/value storage mechanism.
+
+## Tutorial Prerequisites
+
+First, follow [this guide](./install.md) to install Spin v0.9.0 or later. To ensure you have the correct version, you can check with this command:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin --version
+spin 0.9.0
+```
+
+## Creating a New Application
+
+As previously documented, you can go ahead and [create a new Spin application from a template](https://developer.fermyon.com/spin/quickstart#creating-a-new-spin-application-from-a-template). Before you do though, please go ahead and read the [spin template](https://developer.fermyon.com/common/cli-reference#templates) and [spin new](https://developer.fermyon.com/common/cli-reference#new) sections of the Spin Command Line Interface (CLI) documentation. You will learn how to use the commands effectively and also see many handy options to [install](https://developer.fermyon.com/common/cli-reference#install-templates) and [upgrade](https://developer.fermyon.com/common/cli-reference#upgrade-templates) templates and so forth. When you are ready, go ahead and create your new application using commands similar to the ones shown below:
+
+{{ tabs "sdk-type" }}
+
+{{ startTab "Rust"}}
+
+```bash
+spin new http-rust my_http_rust_app
+```
+
+{{ blockEnd }}
+
+{{ startTab "TypeScript" }}
+
+```bash
+spin new http-ts my_http_typescript_app
+```
+
+{{ blockEnd }}
+
+{{ startTab "TinyGo" }}
+
+```bash
+spin new http-go my_http_go_app
+```
+
+{{ blockEnd }}
+
+{{ blockEnd }}
+
+## Configuration
+
+Take special note of the `key_value_stores = ["default"]` line in the `[[component]]` area of the `spin.toml` file, as shown in the next section. A newly scaffolded application will not have this line. You will need to add it.
```

Comment:
Why do they need to add it? Good opportunity to explain per-component access to the store

Reply:
Thanks @kate-goldenring 
This is a really good point, it is covered off in the new push to this branch.

Reply:
Thanks Kate, updates in latest push, and new issues open in dev docs for future articles (concept article etc.)